cmd/get/get.go

```diff
@@ -57,3 +153,101 @@ func toPrintableProjects(projects []*admin.Project) []interface{} {
 
 	return res
 }
+
+func toPrintableTask(tasks []*admin.NamedEntityIdentifier) []interface{} {
+	type printableTasks struct {
+		Name    string `header:"Name"`
+		Project string `header:"Project"`
+		Domain  string `header:"Domain"`
+	}
+
+	res := make([]interface{}, 0, len(tasks))
+	for _, p := range tasks {
+		res = append(res, printableTasks{
+			Name:    p.Name,
+			Domain:  p.Domain,
+			Project: p.Project,
+		})
+	}
+
+	return res
+}
+
+func toPrintableGetTask(tasks []*admin.Task) []interface{} {
+	type printableTask struct {
+		Version    string `header:"Version"`
+		Name    string `header:"Name"`
+		Type  string `header:"Type"`
+		Discoverable  bool `header:"Discoverable"`
+		ResourceType core.ResourceType `header:"ResourceType"`
+	}
+	res := make([]interface{}, 0, len(tasks))
+	for _,task := range tasks {
+		res = append(res,printableTask{
+		Version:    task.Id.Version,
+			Name: task.Id.Name,
+				ResourceType: task.Id.ResourceType,
+				Type : task.Closure.CompiledTask.Template.Type,
+				Discoverable : task.Closure.CompiledTask.Template.Metadata.Discoverable,
+		})
+	}
+	return res
+}
+
+func toPrintableDomain(projects []*admin.Project) []interface{} {
+	type printableDomain struct {
+		Name string `header:"Name"`
+		Id   string `header:"Id"`
+	}
+
+	res := make([]interface{}, 0, len(projects))
+	for _, p := range projects {
+		if p.Name == config.GetConfig().Project {
+			for _, d := range p.Domains {
+				res = append(res, printableDomain{
+					Id:   d.Id,
+					Name: d.Name,
+				})
+			}
+			break
+		}
+	}
+
+	return res
+}
+
+func toPrintableWorkflow(workflows []*admin.NamedEntityIdentifier) []interface{} {
+	type printableWorkflow struct {
+		Name string `header:"Name"`
+		Domain   string `header:"Domain"`
+		Project   string `header:"Project"`
+	}
+
+	res := make([]interface{}, 0, len(workflows))
+	for _, w := range workflows {
+		res = append(res, printableWorkflow{
+			Name:   w.Name,
+			Domain: w.Domain,
+			Project: w.Project,
+		})
+	}
+
+	return res
+}
+
+func toPrintableListWorkflow(workflows []*admin.Workflow) []interface{} {
```

Comment:
Anyway we can change our printing logic to be a bit more generic?
e.g. we can serialize any protobuf into a json object (`map[string]interface{}`) then store somewhere (server side or even just a const here) the list of json paths to use for columns for each object type... then use a library like [this](https://godoc.org/github.com/yalp/jsonpath) to access these fields...

This way you can probably write just `toPrintableObject` and `toPrintableList` functions and have them work on any of the objects we support.

Reply:
I am working on that. I will create a sperate PR for that....This pr is for feedback as product requirement meaning what data we want to present in table